controllers/cloudshell_controller.go

```diff
@@ -138,10 +140,17 @@ func (c *CloudShellReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			if err := c.CreateRouteRule(ctx, cloudshell); err != nil {
 				return ctrl.Result{}, err
 			}
+		}
 
-			// update cloudshell phase to "PhaseCreatedRoute".
-			if err := c.UpdateCloudshellStatus(ctx, cloudshell, cloudshellv1alpha1.PhaseReady); err != nil {
-				return ctrl.Result{}, err
+		// check the pod of job whether is running.
+		if cloudshell.Status.Phase == cloudshellv1alpha1.PhaseCreatedRoute {
+			if ok, err := c.isRunning(ctx, job); !ok {
+				log.Error(err, "cloudshell is not ready, wait to pod running")
```

Comment:
why `log.Error()` here ? 
should we `log.Info()` then `return ctrl.Result{RequeueAfter: time.Duration(3) * time.Second}, nil`